read-fonts/src/tables/postscript/blend.rs

```diff
@@ -0,0 +1,166 @@
+//! Support for the dictionary and charstring blend operator.
+
+use font_types::{BigEndian, F2Dot14, Fixed};
+
+use super::Error;
+use crate::tables::variations::{ItemVariationData, ItemVariationStore, VariationRegionList};
+
+/// The maximum number of region scalars that we precompute.
+const MAX_PRECOMPUTED_SCALARS: usize = 16;
```

Comment:
Why 16?

Reply:
Completely made up number chosen to balance size with trying to capture as many precomputed regions as possible. I'll measure when I have a larger set of CFF2 fonts to test against and adjust accordingly.

Reply:
^ would make a lovely comment :)

Reply:
will do :)